backend/tests/integration/run

```diff
@@ -272,6 +277,28 @@ for suite in "${TEST_SUITES[@]}"; do
         compose_cmd logs > "$tmppath" 2>&1
     fi
 
+    # it was KJ's idea to run tests in a separate session.
+    # for the rest I take the blame. see README.md for the
+    # explanation of how to run your tests separately
+    INITIAL_COMPOSE_FILES="${COMPOSE_FILES}"
+    while read -r; do
+        export PYTEST_TEST_TO_RUN="`basename "$REPLY"`"
+        test_name="${PYTEST_TEST_TO_RUN%%+*}"
+        eval COMPOSE_FILES=\"${INITIAL_COMPOSE_FILES} \${COMPOSE_FILES_$test_name}\"
+        export PYTEST_TEST_TO_RUN="`basename "$REPLY"`"
+        run_tests
+        run_tests_retcode=$?
+        if [ $script_failed -eq 0 ]; then
+            script_failed=$run_tests_retcode
+        fi
+
+        if [ $run_tests_retcode -ne 0 ]; then
+            tmppath=$(TMPDIR=./tests mktemp logs.XXXXXX)
+            echo "-- tests failed, dumping logs to $tmppath"
+            compose_cmd logs >> "$tmppath" 2>&1
+        fi
+    done < <(find "${MENDER_SERVER_PATH}/backend/tests/integration/tests" -name '*+separate.py')
+
```

Comment:
This makes it impossible to parallelize backend integration tests into separate pytest jobs.
Can we:
 * put this in a separate suite (`-s` argument)?
 * put this test in a separate test module and extend the docker-compose.backend-tests-compat.yml to mount this test module under `/tests`?

Reply:
I do not understand. this makes it possible to run any test you put in the test_name+separate.py in a separate session. why do we have to complicate it?